### ISSUE TITLE
Generated code comments omit units and enum names

### ISSUE THREAD

**Issue by hamishwillee** [ArduPilot/pymavlink#220]
Recently we updated the message format to move units and type (enum) information from the message descriptions into specific attributes. 

The generated comments (at least for python) need to be updated to reflect this - the information was removed from the description which is included in code comments.

FYI @amilcarlucas

**Comment by amilcarlucas** [comment]
I fixed it for C and C++ code generation, but I did not update the comments generation for the other generated languages :(

**Comment by hamishwillee** [comment]
@amilcarlucas YOu rock. Thanks! I might do it for Python while I'm playing with it.

**Comment by amilcarlucas** [comment]
Yes, you are correct C, C++, and python are done but the other languages are still missing... reopening
